.github/workflows/run_tests.yml

```diff
@@ -21,27 +21,30 @@ jobs:
     
     strategy:
       matrix:
-       py_version: ["3.6", "3.7", "3.8", "3.9", "3.6-dev", "3.7-dev", "3.8-dev", "3.9-dev"] 
+       py_version: ["3.7", "3.8", "3.9", "3.10", "3.7-dev", "3.8-dev", "3.9-dev", "3.10-dev"]
```

Comment:
Here we can actually remove all the dev versions, because GHA will only collect the latest release version not the actual deve versions. This is an actions bug we discussed earlier I believe.